README.md

```diff
@@ -27,7 +27,7 @@ is designed to be easy to implement and use.
 ## Requirements
 * iOS 8 or later
 * Xcode 7 (iOS 9 SDK)
-* An iZettle SDK Key. Visit [iZettle Developer Page](https://www.izettle.com/se/developer) in order to obtain one.
+* An iZettle SDK Key. Visit [iZettle Developer Page](https://www.izettle.com/gb/developer) in order to obtain one. **Note: SDK keys only work for apps using the same bundle identifier as the one they key were issued for.**
```

Comment:
We might be able to simply this a bit. What do you think about this

```
SDK keys work only for bundle identifiers they were issued for.
```

Reply:
Sure thing, it just took me a while to figure this out and thought it would be a good thing to mention it in the documentation.